upgrade_versions.py

```diff
@@ -217,6 +220,14 @@ def main(args):
         CONF.get('default').get('distro_name'),
         CONF.get('default').get('distro_version'),
         CONF.get('default').get('arch_and_endianness'))
+    committer_name = CONF.get('default').get('committer_name')
+    committer_email = CONF.get('default').get('committer_email')
```

Comment:
This is to make sure the configs are set _before_ actually executing the commands. I think we should find a way of checking all the configs required to execute a given script.
I also don't know if it's a good idea to get the configs directly in the place they are used. This entangles the methods and classes with the config module.
One more point: those configs will also be passed to the function push_new_versions in the next patch.